doc/scripts/genrest.py

```diff
@@ -61,7 +61,7 @@ def expr_str_rst(expr):
 they add to the configuration menus.
 
 The configuration options' information below is extracted directly from
-:program:`Kconfig` using the :file:`~/doc/scripts/genrest.py` script. Click on
+:program:`Kconfig`. Click on
```

Comment:
Could maybe reflow the text too, so that this line isn't much shorter than the one after it.